payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/BoletoTaxIdSpec.kt

```diff
@@ -0,0 +1,27 @@
+package com.stripe.android.ui.core.elements
+
+import androidx.annotation.RestrictTo
+import com.stripe.android.ui.core.R
+import com.stripe.android.uicore.elements.FormElement
+import com.stripe.android.uicore.elements.IdentifierSpec
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.Transient
+
+@Serializable
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP_PREFIX)
+data class BoletoTaxIdSpec(
```

Comment:
Do we need to do any validation of the input here, or limit it to a certain length?